android/src/main/java/com/taplytics/react/TaplyticsReactModule.java

```diff
@@ -243,6 +246,9 @@ public void onNewSession() {
                 params.putBoolean("value", true);
                 sendEvent("newSession", params);
             }
+            public void onError() {
+                
```

Comment:
do we need to send an event with the value of false here?

Reply:
it should also be overriding onError()... 

@Arpit-Patel you can open this file in Android studio and it will show you its errors

Reply:
@VicV why doesn't this one have a promise like the other ones?

Reply:
@jonathannorris I dont know, Adam wrote all of this

Reply:
@jonathannorris there is no promise because its just setting a listener.